src/test/com/kainos/ea/integration/RecruitmentRequestsIntegrationTest.java

```diff
@@ -28,4 +38,36 @@ void getJobs_shouldReturnListOfJobs() {
 
         Assertions.assertTrue(response.size() > 0);
     }
+    @Test
+    void deleteJobRole_shouldReturnJobSuccessfullyDeletedMessage() throws SQLException, DatabaseConnectionException {
+        Job job =new Job(
+                "Marketing Operations Manager",
+                "Develops Software for Kainos",
+                "https://kainossoftwareltd.sharepoint.com/:b:/g/people/EbTM1UOLa0VBvOttkOL3ZNoB0sMjehxvkAaNQEj2dqKMbA?e=iXpeUf",
+                "Experience of building and testing modern software applications",
+                6,
+                "Business Development and Marketing");
+        DatabaseConnector databaseConnector = new DatabaseConnector();
+        Connection c =databaseConnector.getConnection();
+        String sql = "INSERT INTO job_roles (name,description,specification,responsibilities,capability,band_id) VALUES(?,?,?,?,?,?);";
+        PreparedStatement InsertStatement = c.prepareStatement(sql,Statement.RETURN_GENERATED_KEYS);
```

Comment:
Could you separate this into it's own function? Would be cleaner to remove it from here